composer/composer.js

```diff
@@ -69,6 +70,19 @@ exports.Composer = Target.specialize( /** @lends Composer# */ {
         }
     },
 
+    _shawdowRoot: {
+        value: null
+    },
+
+    shawdowRoot: {
+        get: function () {
+            if (!this._shawdowRoot) {
+                this._shawdowRoot = defaultEventManager.getShawdowRootFromNode(this.element);
```

Comment:
Let's re-name getShawdowRootFromNode() to shawdowRootFromNode()

Reply:
ok